lib/galaxy/util/__init__.py

```diff
@@ -570,7 +570,7 @@ def mask_password_from_url( url ):
 
 
 def ready_name_for_url( raw_name ):
-    """ General method to convert a string (i.e. object name) to a URL-ready
+    u""" General method to convert a string (i.e. object name) to a URL-ready
```

Comment:
What is the deal with the unicode `u"text"`? Should we have all docstrings in unicode or only type `u` in front of the ones we think might contain unicode? Does the linter tell us?